src/clanhr/ring_test_client/core.clj

```diff
@@ -5,6 +5,12 @@
   (:use clojure.test
         ring.mock.request))
 
+(defn ->edn [raw]
```

Comment:
Yes that's on the first chapter. Still need to get the rest, but I'll wait for the complete version.